tests/test_msg_tron_signtx.py

```diff
@@ -0,0 +1,698 @@
+# This file is part of the Trezor project.
+#
+# Copyright (C) 2012-2019 SatoshiLabs and contributors
+#
+# This library is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License version 3
+# as published by the Free Software Foundation.
+#
+# This library is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the License along with this library.
+# If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.
+
+import pytest
+
+from trezorlib import debuglink, messages as proto, tron
+from trezorlib.tools import parse_path
+
+from .common import TrezorTest
+
+TRON_DEFAULT_PATH = "m/44'/195'/0'/0/0"
+
+
+def load_device(self):
+    debuglink.load_device_by_mnemonic(
```

Comment:
please take a look at some other tests in the suite and make sure you're using the `@pytest.mark.setup_client` decorator instead of this.

also while you're at it, please remove the class and move all test functions to top level (see e.g. test_msg_stellar_sign_transaction

Reply:
code has been changed to the way `test_msg_stellar_sign_transaction` works